core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -174,3 +185,39 @@ message UpdateEntryResponse {
   // proof contains a proof that the update has been included in the tree.
   GetEntryResponse proof = 1;
 }
+
+// GetMutationsRequest contains the input parameters of the GetMutation APIs.
+message GetMutationsRequest {
+  // epoch specifies the epoch number in which mutations will be returned.
+  int64 epoch = 1;
+  // first_tree_size is the tree_size of the currently trusted log root.
+  // Omitting this field will ommit the log consistency proof from the response.
+  int64 first_tree_size = 2;
+  // page_token defines the starting point for pagination. An empty
+  // value means start from the beginning. A non-empty value requests the next
+  // page of values.
+  string page_token = 3;
+  // page_size is the maximum number of epochs to return.
+  int32 page_size = 4;
+}
+
+// GetMutationsResponse contains the results of GetMutation APIs.
+message GetMutationsResponse {
+  // epoch specifies the epoch number of the returned mutations.
+  int64 epoch = 1;
+  // smr contains the signed map root for the sparse Merkle Tree.
+  trillian.SignedMapRoot smr = 2;
+  // log_root is the latest globally consistent log root.
+  trillian.GetLatestSignedLogRootResponse log_root = 3;
```

Comment:
I'm in the process of updating these fields, see the updates in the [other PR](https://github.com/google/keytransparency/pull/572/files#diff-1945963a3c4b16f704481c78d5d0a38dR148)